linker/Tests/TestCases/IndividualTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using Mono.Linker.Tests.Cases.References.Individual;
+using Mono.Linker.Tests.TestCases;
+using Mono.Linker.Tests.TestCasesRunner;
+using NUnit.Framework;
+
+namespace Mono.Linker.Tests.TestCases
+{
+	[TestFixture]
+	public class IndividualTests
```

Comment:
I don't necessarily love this organization scheme.  But I think it works until we accumulate more tests that need to work like this.